xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/web/DeleteAction.java

```diff
@@ -340,7 +335,13 @@ private Job startDeleteJob(EntityReference entityReference, XWikiContext context
         }
     }
 
-    protected DocumentReferenceResolver<String> getCurrentStringDocumentReferenceResolver()
+    /**
+     * @return the current document reference resolver to be used to resolve a string document into a
+     *         {@link org.xwiki.model.reference.DocumentReference}
+     * @since 14.4.1
+     * @since 14.5RC1
```

Comment:
``@since`` makes sense only for public / protected APIs (i.e. that can be used outside of this class).